internal/schema-registry/command_exporter_status.go

```diff
@@ -0,0 +1,18 @@
+package schemaregistry
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/v3/pkg/config"
+)
+
+func (c *command) newExporterStatusCommand(cfg *config.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "status",
+		Short: "Describe the schema exporter status.",
```

Comment:
I know there's only one subcommand here, but the current description doesn't really make sense. I know we do this for other commands, so up for debate
```suggestion
		Short: "Manage the schema exporter status.",
```

Reply:
Yeah this one's a bit weird. I think I like `manage` more though, so I'll make that change.

Reply:
Yeah this one's a bit weird. I think I like `manage` more though, so I'll make that change.

Reply:
Sweet, let's make that change across the board?